src/applications/personalization/preferences/containers/PreferencesWidget.jsx

```diff
@@ -250,7 +250,7 @@ class PreferencesWidget extends React.Component {
             />
           )}
         </ReactCSSTransitionGroup>
-        <div aria-live="polite">{this.renderContent()}</div>
+        <div>{this.renderContent()}</div>
```

Comment:
I think we might be able to get rid of the extra `<div>` here, so it'd be just `{this.renderContent()}` 👍

Reply:
I _think_ that any function that returns JSX, like the `renderContent` function, needs to return a single root node? Please correct me if I'm wrong about that. But if that's the case we need to either keep this `div`, convert it to a Fragment, or refactor the `renderContent` function to only return a single root node. As it stands it currently can return multiple root JSX elements.